examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<p>"inferring chemical information from residue and atom names and a substructure dictionary distributed with the toolkit" --&gt; "...information from element and connectivity". For example, we could load an ALA from PDB, even if it didn't have a residue name, just as long as it has CONECT records instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>New phrasing is "... for polymers the Toolkit supports inferring chemical information from PDB files. Connectivity can be taken from residue and atom names or CONECT records, and formal charges and bond orders are inferred from a substructure dictionary distributed with the Toolkit."</p>

Reply:
Perfection!